tests/config.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
Ugh, this is cruft generated by pouch, and should not be checked in.  No one should ever have to care what it means or does.  I will have to clean up the repo and  do some research about newer output directory options.  Hopefully this is as solveable as the database file cruft pouch previous dumped all over the place.